content/applications/sales/subscriptions/payment_providers/wire_transfer.rst

```diff
@@ -0,0 +1,151 @@
+
+If :doc:`a bank account <../../../finance/accounting/bank>` has already been defined, the account
+number will be automatically added to the default message generated by Odoo. A bank account can also
+be added afterwards, and the message updated with the appropriate bank information by clicking
+:guilabel:`Reload pending message`.
+
+Once all configuration settings are set, the :icon:`fa-eye-slash` :guilabel:`Unpublished`
+smart button located at the top of the page automatically changes to :icon:`fa-globe`
+:guilabel:`Published` to show that the wire transfer payment option is now visible to customers.
+
+.. tip::
+   Only keep the payment method published if payments are ready to be received. It is possible to
+   keep the payment method enabled, but **not** visible to customers, until payments are ready to be
+   received by setting the smart button to :icon:`fa-eye-slash` :guilabel:`Unpublished` after
+   setting the :guilabel:`State` to :guilabel:`Enabled`.
+
+.. image:: wire_transfer/wire-transfer-published.png
+   :align: center
+   :alt: The wire transfer card enabled and published.
+
+Test mode
+---------
+
+The wire transfer payment method can be tested before it is accessed by customers by using *Test
+Mode*. To enable Test Mode, set the :guilabel:`State` in the payment provider form to
+:guilabel:`Test Mode` and an orange :guilabel:`Test mode` banner appears in the upper-right side of
+the form.
+
+Next, enter the credentials for a bank account with accessible transactions in the
+:guilabel:`Messages` tab. To re-enable normal function after testing, change :guilabel:`State` to
+:guilabel:`Enabled`.
+
+.. note::
+   By default, the payment provider remains **unpublished** in test mode so that it is not visible
+   to visitors.
+
+.. warning::
+   It is recommended to use test mode on a duplicate or a test database to avoid potential issues
+   with invoice numbering.
+
+.. _wire_transfer/payment:
+
+Pay by wire transfer
+====================
+
+With wire transfers :ref:`enabled and published <wire_transfer/configuration>`, customers will see
+wire transfers as an available payment method at checkout when purchasing a product. When this
+method is selected by a customer, they are shown the *Pending Message* set in the :ref:`wire
+transfer configuration page <wire_transfer/configuration>` after confirming their purchase. Once
+the purchase is confirmed, a sales order is created.
+
+.. image:: wire_transfer/payment-instructions-checkout.png
+   :align: center
+   :alt: The wire transfer pending message shown to customers.
+
+.. important::
+   Online orders remain in the *Quotation sent* (i.e. unpaid order) stage until both the payment is
+   received, *and* the order confirmed manually using the :guilabel:`Confirm` button on the sales
+   order.
+
+To begin processing the payment, navigate to :menuselection:`Sales app --> Orders --> Orders` and
+click the corresponding sales order. A green :guilabel:`In Progress` banner appears in the
```

Comment:
During my run-through, I wasn't seeing this 'In Progress' banner -- might be worth a quick double-check during your revisions.